transfer-products/cloudformation.yml

```diff
@@ -76,8 +76,6 @@ Resources:
       Role: !GetAtt Role.Arn
       Runtime: python3.9
       Timeout: 900
-      EphemeralStorage:
```

Comment:
Ah, I'm assuming we no longer need this now that we're copying objects bucket-to-bucket?